graphql2/schema.graphql

```diff
@@ -972,7 +973,8 @@ input CalcRotationHandoffTimesInput {
   handoff: ISOTimestamp!
   from: ISOTimestamp
   timeZone: String!
-  shiftLengthHours: Int!
+  shiftLength: Int!
+  shiftType: RotationType!
```

Comment:
We can't make breaking API changes; there is already code in the wild using `shiftLengthHours` -- we *can* make `shiftLengthHours` optional and add a new field, however.

I think using `ISODuration` (`shiftLength: ISODuration`) would be better than adding two fields, since it already supports encoding hour vs day vs month vs etc.

TBH, it might make sense to look at deprecating (at least from UI/API) the concept of a "rotation type" and instead just treat it as an ISODuration